horace_core/sqw/@sqw/signal.m

```diff
@@ -0,0 +1,46 @@
+function wout=signal(w,name)
+% Set the intensity of an sqw object to the values for the named argument
```

Comment:
as noted in sqw.m, this sounds like a function that returns a signal. Set_signal may be more appropriate

From the argument description below, it looks as if this is a special use to allow the input parameters to be visualised. Saying what this usage is explicitly would be helpful

Reply:
Yes, it can be used to visualise the input parameters, but actually, its main use is for corrections. For example, the intensity of magnetic excitations are often approximately proportional to the energy transfer, so that high energy excitations can appear much fainter. So, sometimes you might want to analyse S(Q,E)/E rather than S - at the moment (because this function was removed from Horace 3.6 but was in 3.5) you have to do something `wcorr = w / sqw_eval(w, @(h,k,l,e) e)` - whereas before you could do `wcorr = w / w.signal('E')` - or to be more correct you would use the Boltzmann thermal occupation factor `wcorr = w / exp(-w.signal('E')/temperature)`. For phonon signals, you might want to correct for the `Q^2` dependence with e.g. `wcorr = w / w.signal('Q').^2`

Reply:
As we are recovering old function, mentioned elsewhere and used in other peoples code I can not easy change its name. 
A separate ticket deprecating `signal` and introducing `set_signal` would be easy to write and implement if the stakeholders approve it.